DateParameter unable to parse pattern correctly

### Current Behavior

I just upgraded from 8.0.3. to 9.0.0, I get the following exception:
`org.frankframework.core.ListenerException: Pipe [call xxx] caught exception: exception calling JavaListener [xxx]: Pipe [checkTime] got exception on transformation: Cannot parse [{0,time,HH:mm}]: (IllegalArgumentException) Cannot format given Object as a Date`

It occurs in the following pipe:

```       
        <pipe name="checkTime"
              className="org.frankframework.pipes.XmlSwitch"
              styleSheetName="Uploadxxx/xsl/checkTime.xsl"
              getInputFromFixedValue="&lt;dummy/&gt;">
            <param name="time" type="TIME" pattern="{now,time,HH:mm}"/>
            <forward name="doTomorrow" path="setTommorowDate"/>
            <forward name="doNormal" path="insertYear"/>
        </pipe>
```

Where before SimpleDateFormat.parse was called, it is now SimpleDateFormat.format.
Where the former returns an Date, latter returns a String.

When MessageFormat.format is called in AbstractParameter.formatPattern(), the String value is sent as a byte array. It seems not be able to handle it.

See below my debugger:
![Image](https://github.com/user-attachments/assets/36b1cb70-fb29-49d3-a39a-46f1f42f3c94)

### Expected Behavior

No exception is thrown and it will format correctly.

### Environment Information

9.0.0 on Tomcat 10.1 running with JDK 17 on a Windows machine.

### Steps To Reproduce

_No response_

### Configuration

```xml
<pipe name="checkTime"
              className="org.frankframework.pipes.XmlSwitch"
              styleSheetName="Uploadxxx/xsl/checkTime.xsl"
              getInputFromFixedValue="&lt;dummy/&gt;">
            <param name="time" type="TIME" pattern="{now,time,HH:mm}"/>
            <forward name="doTomorrow" path="setTommorowDate"/>
            <forward name="doNormal" path="insertYear"/>
        </pipe>
```

### Input

_No response_

### What database are you using?

_No response_

### What browsers are you seeing the problem on?

_No response_

### Relevant Log Output

```console
2025-03-14 17:36:55,184 ERROR [UploadxxxDirectoryListener-listener[1]] {} receivers.Receiver - Receiver [UploadxxxDirectoryListener] caught Exception processing message
org.frankframework.core.ListenerException: Pipe [call UploadxxxFile] caught exception: exception calling JavaListener [UploadxxxFile]: Pipe [checkTime] got exception on transformation: Cannot parse [{0,time,HH:mm}]: (IllegalArgumentException) Cannot format given Object as a Date
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:621) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.Receiver.processMessageInAdapter(Receiver.java:1286) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.Receiver.processRawMessage(Receiver.java:1126) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.Receiver.processRawMessage(Receiver.java:1086) [frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.PullingListenerContainer$ListenTask.run(PullingListenerContainer.java:313) [frankframework-core-9.0.0.jar:9.0.0]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.base/java.lang.Thread.run(Thread.java:840) [?:?]
Caused by: org.frankframework.core.PipeRunException: Pipe [call UploadxxxFile] caught exception: exception calling JavaListener [UploadxxxFile]: Pipe [checkTime] got exception on transformation: Cannot parse [{0,time,HH:mm}]: (IllegalArgumentException) Cannot format given Object as a Date
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:397) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:37) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeProcessor.execute(TransactionAttributePipeProcessor.java:97) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:46) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:68) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:189) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:114) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:227) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:162) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:45) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:217) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:127) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:60) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:49) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:42) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:41) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:44) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.core.PipeLine.process(PipeLine.java:413) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:613) ~[frankframework-core-9.0.0.jar:9.0.0]
	... 7 more
Caused by: org.frankframework.core.SenderException: exception calling JavaListener [UploadxxxFile]: Pipe [checkTime] got exception on transformation: Cannot parse [{0,time,HH:mm}]: (IllegalArgumentException) Cannot format given Object as a Date
------ trace was to long  -----
Caused by: java.lang.IllegalArgumentException: Cannot format given Object as a Date
	at java.base/java.text.DateFormat.format(DateFormat.java:342) ~[?:?]
	at java.base/java.text.Format.format(Format.java:159) ~[?:?]
	at java.base/java.text.MessageFormat.subformat(MessageFormat.java:1353) ~[?:?]
	at java.base/java.text.MessageFormat.format(MessageFormat.java:887) ~[?:?]
	at java.base/java.text.Format.format(Format.java:159) ~[?:?]
	at java.base/java.text.MessageFormat.format(MessageFormat.java:862) ~[?:?]
	at org.frankframework.parameters.AbstractParameter.formatPattern(AbstractParameter.java:539) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.parameters.AbstractParameter.getValue(AbstractParameter.java:376) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:89) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugParameterResolvedTo(IbisDebuggerAdvice.java:448) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.parameters.DateParameter$$SpringCGLIB$$0.getValue(<generated>) ~[?:?]
	at org.frankframework.parameters.ParameterList.getValue(ParameterList.java:162) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.parameters.ParameterList.getValues(ParameterList.java:133) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.pipes.XmlSwitch.doPipe(XmlSwitch.java:131) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:37) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeProcessor.execute(TransactionAttributePipeProcessor.java:97) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:46) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:68) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:189) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:114) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:227) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:162) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:45) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:217) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:127) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:60) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:49) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:42) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:41) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:44) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.core.PipeLine.process(PipeLine.java:413) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:613) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.Receiver.processMessageInAdapter(Receiver.java:1286) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.Receiver.processRequest(Receiver.java:1060) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.JavaListener.processRequest(JavaListener.java:175) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.receivers.JavaListener.processRequest(JavaListener.java:152) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.senders.IbisLocalSender.sendMessage(IbisLocalSender.java:283) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderInputOutputAbort(IbisDebuggerAdvice.java:233) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderSendMessage(IbisDebuggerAdvice.java:290) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.senders.IbisLocalSender$$SpringCGLIB$$0.sendMessage(<generated>) ~[?:?]
	at org.frankframework.pipes.MessageSendingPipe.sendMessage(MessageSendingPipe.java:658) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.pipes.MessageSendingPipe.sendMessageWithRetries(MessageSendingPipe.java:436) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:369) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:37) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeProcessor.execute(TransactionAttributePipeProcessor.java:97) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:46) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:68) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:189) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:114) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:227) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:162) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:45) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:217) ~[frankframework-core-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:102) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:127) ~[frankframework-ladybug-debugger-9.0.0.jar:9.0.0]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.1.jar:6.2.1]
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(<generated>) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:60) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:49) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:42) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:41) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:44) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.core.PipeLine.process(PipeLine.java:413) ~[frankframework-core-9.0.0.jar:9.0.0]
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:613) ~[frankframework-core-9.0.0.jar:9.0.0]
	... 7 more
```

### Anything else?

_No response_

--- a/core/src/main/java/org/frankframework/parameters/AbstractParameter.java
+++ b/core/src/main/java/org/frankframework/parameters/AbstractParameter.java
@@ -633,7 +633,7 @@ public abstract class AbstractParameter implements IConfigurable, IWithParameter
 			String namelc=name.toLowerCase();
 			switch (namelc) {
 				case "now":
-					if ("date".equals(formatType)) {
+					if ("date".equalsIgnoreCase(formatType) || "time".equalsIgnoreCase(formatType)) {
 						substitutionValue = new Date();
 					} else{
 						substitutionValue = formatDateToString(new Date(), formatString);

--- a/core/src/test/java/org/frankframework/parameters/DateParameterTest.java
+++ b/core/src/test/java/org/frankframework/parameters/DateParameterTest.java
@@ -418,4 +418,87 @@ public class DateParameterTest {
 		assertEquals(date, result);
 	}
 
+	@Test
+	public void testTimeParameterWithTimePatternWithoutFormat() throws Exception {
+
+		// Arrange
+		DateParameter parameter = new DateParameter();
+		parameter.setName("time");
+		parameter.setFormatType(DateFormatType.TIME);
+		parameter.setPattern("{now,time}"); // Does not work if you do not specify 'time' in the pattern. Seems like a bug to me, inconsistent with how DateFormatType.DATE works
+		parameter.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+		PipeLineSession session = new PipeLineSession();
+
+		// Act
+		Object result = parameter.getValue(alreadyResolvedParameters, message, session, true);
+
+		// Assert
+		assertInstanceOf(Date.class, result);
+	}
+
+	@Test
+	public void testTimeParameterWithTimePatternWithFormat() throws Exception {
+
+		// Arrange
+		DateParameter parameter = new DateParameter();
+		parameter.setName("time");
+		parameter.setFormatType(DateFormatType.TIME);
+		parameter.setPattern("{now,time,HH:mm:ss}"); // Does not work if you do not specify 'time' in the pattern. If you specify a format, it HAS to be this format.
+		parameter.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+		PipeLineSession session = new PipeLineSession();
+
+		// Act
+		Object result = parameter.getValue(alreadyResolvedParameters, message, session, true);
+
+		// Assert
+		assertInstanceOf(Date.class, result);
+	}
+
+	@Test
+	public void testDateParameterWithDatePatternWithoutFormat() throws Exception {
+
+		// Arrange
+		DateParameter parameter = new DateParameter();
+		parameter.setName("date");
+		parameter.setFormatType(DateFormatType.DATE);
+		parameter.setPattern("{now}"); // Specifying just 'date' here does not work. Not consistent with DateFormatType.TIME
+		parameter.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+		PipeLineSession session = new PipeLineSession();
+
+		// Act
+		Object result = parameter.getValue(alreadyResolvedParameters, message, session, true);
+
+		// Assert
+		assertInstanceOf(Date.class, result);
+	}
+
+	@Test
+	public void testDateParameterWithDatePatternWithFormat() throws Exception {
+
+		// Arrange
+		DateParameter parameter = new DateParameter();
+		parameter.setName("date");
+		parameter.setFormatType(DateFormatType.DATE);
+		parameter.setPattern("{now,date,yyyy-MM-dd}"); // Specifying full date format makes it work, but it HAS to be this format
+		parameter.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+		PipeLineSession session = new PipeLineSession();
+
+		// Act
+		Object result = parameter.getValue(alreadyResolvedParameters, message, session, true);
+
+		// Assert
+		assertInstanceOf(Date.class, result);
+	}
 }

--- a/core/src/test/java/org/frankframework/parameters/ParameterTest.java
+++ b/core/src/test/java/org/frankframework/parameters/ParameterTest.java
@@ -1523,4 +1523,45 @@ public class ParameterTest {
 
 		assertEquals("hallo", p.getValue(alreadyResolvedParameters, input, session, false));
 	}
+
+	@Test
+	public void testParameterWithTimePattern() throws Exception {
+
+		// Arrange
+		Parameter parameter = new Parameter();
+		parameter.setName("time");
+		parameter.setPattern("{now,time,HH:mm}");
+		parameter.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+		PipeLineSession session = new PipeLineSession();
+
+		// Act
+		Object result = parameter.getValue(alreadyResolvedParameters, message, session, true);
+
+		// Assert
+		assertInstanceOf(String.class, result);
+		assertTrue(((String)result).matches("\\d{1,2}:\\d{2}"));
+	}
+
+	@Test
+	public void testParameterWithDatePattern() throws Exception {
+
+		// Arrange
+		Parameter parameter = new Parameter();
+		parameter.setName("time");
+		parameter.setPattern("{now,date,HH:mm}");
+		parameter.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+		PipeLineSession session = new PipeLineSession();
+
+		// Act
+		Object result = parameter.getValue(alreadyResolvedParameters, message, session, true);
+
+		// Assert
+		assertTrue(((String)result).matches("\\d{1,2}:\\d{2}"));
+	}
 }

